sympy/stats/tests/test_stochastic_process.py

```diff
@@ -0,0 +1,36 @@
+from sympy import (S, symbols, FiniteSet, Eq, Matrix, MatrixSymbol, Float)
+from sympy.stats import DiscreteMarkovChain, P
+from sympy.stats.rv import RandomIndexedSymbol
+from sympy.stats.symbolic_probability import Probability
+from sympy.utilities.pytest import raises
+
+def test_DiscreteMarkovChain():
+
+    # pass only the name
+    X = DiscreteMarkovChain("X")
+    assert X.is_Discrete == True
+    assert X.is_Continuous == False
+    assert X.state_space == S.Reals
```

Comment:
The above seems contradictory. If the state-space is Reals then I'd expect this to be "continuous"

Reply:
> This was for time.

+1